dev/fattest.simplicity/src/componenttest/topology/database/container/DatabaseContainerFactory.java

```diff
@@ -187,4 +187,21 @@ private static boolean isJdbcDriverAvailable(DatabaseContainerType type) {
 
         return result;
     }
+
+    /**
+     * Creates a container timeout duration (in minutes) based on where the test is being run.
+     *
+     * @param  low  - For fast systems: typically your local system
+     * @param  high - For slow systems: typically our build systems
+     *
+     * @return      The timeout duration
+     */
+    private static Duration getContainerTimeout(int low, int high) {
```

Comment:
I'm not a huge fan of `low` and `high`, since to me that sounds like a range. I'd prefer `short` and `long`, but those are reserved. Maybe `shortTimeout` and `longTimeout`? or `fast`/`fastTimeout` and `slow`/`slowTimeout`? I'm not super concerned, so if you want to leave it as `low` and `high` that's fine, just a couple of other ideas.